### ISSUE TITLE
Feature request: Profile images

### ISSUE THREAD

**Issue by egasz** [Wingysam/Christmas-Community#162]
Would it be possible to add a button on the profile that would allow the user to upload a new image to his/her profile from a local folder?

**Comment by cj13579** [comment]
Handling that is described briefly [in the README](https://github.com/Wingysam/Christmas-Community?tab=readme-ov-file#default-profile-pictures).

You can place the images anywhere in `static/img` including adding new folders and then just start the path, when you change it on the profile, to `/img/me.png` (or whatever). If you have images that are publicly available online, like [your GitHub profile image](https://avatars.githubusercontent.com/u/7276831?v=4), you can also use full URLs on the profile.

**Comment by egasz** [comment]
Thank you for your reply, that's exactly what I did to change my profile picture, but my request was for other members of my family (other users) that wanted to change their pics and they don't have access to the backend, so a button to upload their own pics would be nice. But thank you guys for all your work!

**Comment by cj13579** [comment]
Yeah, OK. So, the ability to upload an image from the front end. That makes sense!

**Comment by cj13579** [comment]
For now, you _can_ set their photo for them. Not ideal though.
